cmd/kubeadm/app/phases/copycerts/copycerts.go

```diff
@@ -92,7 +95,7 @@ func UploadCerts(client clientset.Interface, cfg *kubeadmapi.InitConfiguration,
 		return err
 	}
 
-	secretData, err := getSecretData(cfg, decodedKey)
+	secretData, err := getDataFromInitConfig(cfg, decodedKey)
```

Comment:
This functions gets the certs from disk, can we use another name?

Reply:
Yeah... the current name is not that good. I will update when I do a final smoke testing.

Reply:
Thanks :)